.github/workflows/release.yml

```diff
@@ -42,8 +42,8 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: yarn run publish && deno install --no-lock && deno publish
-          version: yarn run version:npm && yarn run version:jsr
+          publish: yarn run publish
+          version: yarn run version
```

Comment:
It looks like changesets expects these to be a single command, rather than multiple scripts. So I've combined these into individual `package.json` scripts

Reply:
I see! Thanks.